index.js

```diff
@@ -268,6 +270,87 @@ const createClient = (profile, userAgent, opt = {}) => {
 		};
 	};
 
+	const bestPrices = async (from, to, opt = {}) => {
+		from = profile.formatLocation(profile, from, 'from');
+		to = profile.formatLocation(profile, to, 'to');
+
+		opt = Object.assign({
+			via: null, // let journeys pass this station?
+			transfers: -1, // maximum nr of transfers
+			transferTime: 0, // minimum time for a single transfer in minutes
+			bike: false, // only bike-friendly journeys
+			tickets: false, // return tickets?
+			polylines: false, // return leg shapes?
+			subStops: false, // parse & expose sub-stops of stations?
+			entrances: false, // parse & expose entrances of stops/stations?
+			remarks: true, // parse & expose hints & warnings?
+			scheduledDays: false, // parse & expose dates each journey is valid on?
+		}, opt);
+		if (opt.via) {
+			opt.via = profile.formatLocation(profile, opt.via, 'opt.via');
+		}
+
+		let when = new Date();
+		if (opt.departure !== undefined && opt.departure !== null) {
+			when = new Date(opt.departure);
+			if (Number.isNaN(Number(when))) {
+				throw new TypeError('opt.departure is invalid');
+			}
+			const now = new Date();
+			let today = DateTime.fromObject({
```

Comment:
Sorry, even though I told you to do it this way, I *think* this isn't correct either: For example, if `profile.timezone` is `Europe/Berlin`, the timezone-naive `now` is `2024-05-06T00:30+01:00` because it runs on a machine in `Europe/London`, then this will create a Berlin-timezone timestamp for the next day.

I can think of a least two options, the latter being the more straightforward one IMO:

1. Change `now` to be timezone-aware, so that `now.{year, month,day}` are correct.
2. Just use one chain of `DateTime` calls, as illustrated below.

Also, while implicitly casting `today` to a `number` below works, it would be cleaner to add `.toMillis()`.

```js
const today = DateTime
	.fromMillis(Date.now(), {
		zone: profile.timezone,
		locale: profile.locale,
	})
	.startOf('day')
	.toMillis()
```